pnpbridge/src/adapters/src/impinj_reader_r700/helpers/string_manipulation.h

```diff
@@ -0,0 +1,11 @@
+#include <string.h>
+
+typedef struct Str_Trim_Data {
```

Comment:
>Str_Trim_Data [](start = 15, length = 13)

Curious why we don't use inbuilt string type here?

Reply:
Because I'm not familiar with the conventions!  Happy to take any suggestions here.  I guess I was thinking you could save the step of using "strlen()" externally if you just return the length with the string.

Reply:
fixed in a508473